include/zephyr/bluetooth/gatt.h

```diff
@@ -167,16 +167,57 @@ typedef ssize_t (*bt_gatt_attr_write_func_t)(struct bt_conn *conn,
 
 /** @brief GATT Attribute structure. */
 struct bt_gatt_attr {
-	/** Attribute UUID */
+	/** @brief Attribute Type, aka. "UUID" */
 	const struct bt_uuid *uuid;
-	/** Attribute read callback */
+
+	/** @brief Attribute Value function
```

Comment:
```suggestion
	/** @brief Attribute Read Value function
```

Reply:
"Attribute Value" is a term in the specification. I think we should use it.

Reply:
Based on [the other discussion](https://github.com/zephyrproject-rtos/zephyr/pull/76210#discussion_r1687893549), I'll make this be "Attribute Value read method".